linkerd/README.md

```diff
@@ -12,6 +12,10 @@ This integration sends your Linkerd metrics to Datadog, including application su
 
 ## Setup
 
+This OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][16].
```

Comment:
This one uses `openmetrics_endpoint` as in https://github.com/DataDog/integrations-core/pull/15073#discussion_r1317050535.

```suggestion
This OpenMetrics-based integration has a latest mode (enabled by setting `openmetrics_endpoint` to point to the target endpoint) and a legacy mode (enabled by setting `prometheus_url` instead). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][16].
```